indico/web/templates/layout/base.html

```diff
@@ -1,36 +1,38 @@
 {% block page %}
     <div class="{% block page_class %}management-page{% endblock %}"
          {% if self.page_id()|trim %}id="{% filter trim %}{% block page_id %}{% endblock %}{% endfilter %}"{% endif %}>
-        {% if self.title() %}
+        {% if self.title() or self.description() or self.info() %}
```

Comment:
Should we really use that base layout within a dialog? This solution looks a bit like stretching `layout/base.html` too much, since it's clearly intended for pages. We could have another base template `layout/dialog.html` for dialogs and then make sure via CSS that the style of both match.